prober/prober.go

```diff
@@ -29,6 +30,17 @@ import (
 // in memory.
 const recentHistSize = 10
 
+type ClassType string
+
+var (
+	ClassDERPUDP    ClassType = "derp_udp"
+	ClassDERPMesh   ClassType = "derp_mesh"
+	ClassDERPMap    ClassType = "derp_map"
+	ClassTLS        ClassType = "tls"
+	ClassQueueDepth ClassType = "derp_qd"
+	ClassBandwidth  ClassType = "derp_bw"
+)
```

Comment:
These are only used once since we don't need to conditionalize to tack on extra labels, they can be removed/reverted to string.